t/secret/aws.t

```diff
@@ -314,3 +314,5 @@ GET /t
     }
 --- response_body
 all done
+--- error_log
```

Comment:
The testing of these changes is an expected change. These tests have an admin api call to the consumer and use a non-existent secret reference, which in the new logic triggers a secret lookup failure and an error log. It didn't break the original test path.